observatory-platform/observatory/platform/utils/release_utils.py

```diff
@@ -0,0 +1,449 @@
+# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import re
+import pendulum
+from observatory.platform.utils.api import make_observatory_api
+from observatory.platform.utils.config_utils import find_schema
+from typing import List, Union, Tuple
+from observatory.api.client.model.dataset_release import DatasetRelease
+from observatory.api.client.model.dataset import Dataset
+import datetime
+
+
+def address_to_gcp_fields(address: str) -> Tuple[str, str, str]:
+    """Take an address in the DatasetStorage field, and return the project ID, dataset ID, table ID.
+
+    :param address: Address string in the DatasetStorage field.
+    :return GCP project, dataset, table.
+    """
+
+    return tuple(address.split("."))
```

Comment:
It's currently unused since that same information is duplicated in several different places, and there is also very tight coupling with gcp. However, in the event that the information duplication is removed, or telescopes start to be written that are more cloud platform agnostic, the service and address fields can be used with parsers like this to retrieve the table URI information.